sass/components/html/listing--navigation.html

```diff
@@ -3,9 +3,10 @@
     <!-- Title with additional text, links, and grandchilds section item. -->
     <div class="section__item">
       <div class="listing listing--navigation">
-        <a href="#" class="listing__item-link"></a>
-        <h2 class="listing__section-title">Title with additional text, links and grandchilds</h2>
-        <p class="listing__description">This is the description. EU strategy, its role in setting priorities, and its implementation through EU policy.</p>
+        <a href="#" class="listing__item-link" title="Title with additional text, links and grandchilds">
```

Comment:
@nelsongrin , i'm confused....why did you put the "old" listing approach back, here..?